### DISCUSSION TITLE
Using Crazyflies without a motion capture system.

### DISCUSSION THREAD

**Discussion by WoodyloveBo** [bitcraze/discussions#1029] [Q&A]
I'm studying a quadrotor with Crazyflie. We're doing a basic update to Crazyflies and Crazyradio 2.0 at bitcraze, and running them using cfclient's Python script. (Without a motion capture system) But Crazyflies didn't work. Is there a way to make Crazyflies work without a motion capture system? (excluding mobile phone apps and Xbox)

**Comment by krichardsson** [comment]
A mocap system is not required.

If you want to fly the Crazyflie autonomously in some way, for instance from a python script, you need some sort of positioning support though. This could for instance be a mocap system (on the expensive side) but it could also be the Loco system, the Lighthouse system or a flow deck.
